src/app/containers/BlockChain/Web3/saga.ts

```diff
@@ -1,24 +1,37 @@
 
 
+import { toast } from "react-toastify";
 import { call, put, select, takeLatest } from "redux-saga/effects";
-import { injected } from "app/containers/BlockChain/utils/wallet/connectors";
+import { CONNECTORS } from "./constants";
 import { selectWeb3Domain } from "./selectors";
 import { Web3Actions } from "./slice";
-import { Web3State } from "./types";
+import { ConnectorPayload, Web3State } from "./types";
 
-export function* connectToWallet() {
+export function* connectToWallet(action: { type: string, payload: ConnectorPayload }) {
+  const { walletName } = action.payload
+  const connector = CONNECTORS[walletName]
+  if (walletName === 'Coin 98') {
```

Comment:
'Coin 98' should be a a constant

Reply:
it is
it's a type, containing valid wallet names, we can not enter an invalid name